client-course-schedulizer/src/components/Toolbar/Searchbar/Searchbar.scss

```diff
@@ -0,0 +1,3 @@
+.searchbar {
+  width: 300px;
+}
```

Comment:
I tried to do this multiple times and it didn't work. I'm not really sure what's different now, but I'm glad it's working.